resources/conf/log_config.toml

```diff
@@ -29,6 +29,11 @@ enable = false
 logfile = "/dev/stdout" # This file will be created inside router container.
 logType = "text" # LogTypes can be "text" or "json"
 
+[accessLogs.excludes]
```

Comment:
Shall we add this to the templates file too ?

Reply:
For log_config.toml we don't have a template file

Reply:
Let's add a comment then similar to the 
https://github.com/wso2/product-microgateway/pull/3430/files#diff-757ae11db4f6dc0be3215293e11e52484ab29526d0a15f884fc3f67f50630730L25
and
https://github.com/wso2/product-microgateway/pull/3430/files#diff-757ae11db4f6dc0be3215293e11e52484ab29526d0a15f884fc3f67f50630730L14

With another PR.